### ISSUE TITLE
[BUG] CI crashing for Ubuntu,3.7,pyqt5 -- Pyinstaller cannot find module matplotlib

### ISSUE THREAD

**Issue by jbutle55** [wkentaro/labelme#758]
CI actions in the last 3 days have continuously crashed for run Ubuntu-latest,3.7,pyqt5. Pyinstaller cannot seem to locate matplotlib. Possibly matplotlib needs to be installed with an older version? Maybe 3.3.0 or earlier? Can be seen in CI https://github.com/wkentaro/labelme/actions/runs/208870714 and https://github.com/wkentaro/labelme/actions/runs/214657163

Traceback:
  File "/usr/share/miniconda/envs/test/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "__init__.py", line 26, in <module>
  File "/usr/share/miniconda/envs/test/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "testing.py", line 4, in <module>
  File "/usr/share/miniconda/envs/test/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "imgviz/__init__.py", line 18, in <module>
  File "/usr/share/miniconda/envs/test/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 493, in exec_module
    exec(bytecode, module.__dict__)
  File "imgviz/depth.py", line 1, in <module>
ModuleNotFoundError: No module named 'matplotlib'
##[error]Process completed with exit code 255.

**Comment by nkuhzx** [comment]
The author uses the file **labelme.spec** to package the project. In the original file,'excludes=[matplotlib]', I changed it to, **'excludes=[]**' because 'matplotlib' is used in the imgviz package. After using the command '**pyinstaller labelme.spec**' to build standalone executable, you can run the executable file without error.

**Comment by jbutle55** [comment]
@nkuhzx is correct, I found that also corrected the failures for me. I had to specify a matplotlib version older than 3.3.0 however due to the MATPLOTLIBDATA environment variable being removed in 3.3.0. 

I included the changes to correct these CI failures in pull request https://github.com/wkentaro/labelme/pull/763 along with the added features I had originally been working on when I discovered the CI failures.
